src/main/resources/helptexts/accessRights.html

```diff
@@ -1,5 +1,37 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Access rights</h1>
+<p>Description of the accessibility to the dataset files.</p>
+<p>The dataset files may be made accessible to users and/or re-users in various ways. The following
+    categories are available:</p>
+<ul>
+    <li><h5>Open access</h5>
+        <p>The dataset files are accessible to all users of EASY and 'CC0 Waiver - No Rights
+            Reserved' applies.
+            For more information please visit https://creativecommons.org/about/cc0.
+            In this category all possible rights (such as copyrights and database rights) on the
+            dataset files have been waived.</p>
+    </li>
+    <li><h5>Open access for registered users</h5>
```

Comment:
Only `Open Access` and `Restricted Access` are used in this deposit-form. Please remove the other options